android/src/app/organicmaps/car/screens/MapScreen.java

```diff
@@ -91,16 +91,25 @@ private Item createBookmarksItem()
 
   private void openSearch()
   {
+    // Details in UiHelpers.createSettingsAction()
```

Comment:
This looks like a bug...

Reply:
Check the answer in [android/src/app/organicmaps/car/util/UiHelpers.java](https://github.com/organicmaps/organicmaps/pull/5045/files/833879d9d1d09a5e41ac1fc8840703e27dd7085e#diff-49a3cc630c3be3988531c49a1f8d736da45d3fce38f380433ebe340d79602931)